app/parse.py

```diff
@@ -8,9 +18,54 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag.text for tag in quote_soup.select(".tag")]
+    )
+
+
+def get_next_page(page_soup: BeautifulSoup) -> str:
+    next_page = page_soup.select_one(".pager > li.next > a")
+    if next_page:
+        return next_page["href"]
```

Comment:
The function `get_next_page` should explicitly return `None` if `next_page` is not found. This ensures that the function always returns a string or `None`, which is important for clarity and avoiding potential issues in the `get_quotes` loop.